api/swagger.yaml

```diff
@@ -1923,6 +1923,38 @@ definitions:
                 type: "array"
                 items:
                   type: "string"
+          Secrets:
+            description: "Secrets contains references to zero or more secrets that will be exposed to the container."
```

Comment:
I think `exposed to the service` is better than `exposed to the container`, WDYT?